Zend/zend_closures.c

```diff
@@ -51,9 +53,12 @@ ZEND_METHOD(Closure, __invoke) /* {{{ */
 		Z_PARAM_VARIADIC_WITH_NAMED(args, num_args, named_args)
 	ZEND_PARSE_PARAMETERS_END();
 
-	if (call_user_function_named(CG(function_table), NULL, ZEND_THIS, return_value, num_args, args, named_args) == FAILURE) {
-		RETVAL_FALSE;
-	}
+	zend_fcall_info_cache fcc = {
```

Comment:
Nit: This causes some unnecessary initialization or most fields that are set later. It's probably elided if `zend_closure_get_closure` is inlined though.